boards/xtensa/intel_adsp_cavs15/tools/flash.sh

```diff
@@ -13,7 +13,7 @@ if [ -z "$2" ]
 elif [ -n "$3" ] && [ -n "$4" ]
   then
     echo "Signing with key " $key
-    west sign -d ${BUILD} -t rimage -p $4 -D $3 -- -k $2
+    west sign -d ${BUILD} -t rimage -p $4 -D $3 -- -k $2 no-manifest
```

Comment:
Shouldn't this be `--no-manifest` as a proper command-line parameter?

Reply:
done